addons/l10n_dk/models/account_move.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models, fields, api
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    currency_rate_at_transaction = fields.Float(
+        string='Rate',
+        compute='_compute_currency_rate_at_transaction', readonly=True, store=True,
+        digits=(12, 6),
+    )
+
+    @api.depends('currency_id')
```

Comment:
We depends only on `currency_id` but we probably need to depends on several staff (as the company, the `invoice_date` and the `date`)